supervisor/task.go

```diff
@@ -8,6 +8,7 @@ import (
 
 // StartResponse is the response containing a started container
 type StartResponse struct {
+	Pid       int
```

Comment:
nit: can we call this `ExecPid` since it won't be filled in the case of a newly created container.